protocol/0078-NWLI-network_wide_limits.md

```diff
@@ -0,0 +1,128 @@
+* When the `limits.markets.maxLimitOrders` limit is reached in the market, but `limits.markets.maxPeggedOrders` is not active yet it is still possible to successfully submit a pegged order (<a name="0078-NWLI-026" href="#0078-NWLI-026">0078-NWLI-026</a>).
+* When the `limits.markets.maxLimitOrders` limit is reached in the market, but `limits.markets.maxLPs` is not active yet it is still possible to successfully submit a new liquidity provision order (<a name="0078-NWLI-027" href="#0078-NWLI-027">0078-NWLI-027</a>).
+
+### Pegged orders
+
+* Attempt to place an additional pegged order by a party already active in the market when the number of pegged orders on the book is equal to `limits.markets.maxPeggedOrders` results in a rejection. Error message attributes the rejection to the limit. (<a name="0078-NWLI-011" href="#0078-NWLI-011">0078-NWLI-011</a>)
+* Reaching a limit in one of the markets doesn't affect the ability to place the orders in other markets which are still below the limit. (<a name="0078-NWLI-012" href="#0078-NWLI-012">0078-NWLI-012</a>)
+* Lowering `limits.markets.maxPeggedOrders` to the number number of pegged orders on the book minus 2 results in no changes to the order book composition. Once 2 of the orders get filled and 1 gets cancelled it is possible to successfully submit one additional pegged order (<a name="0078-NWLI-013" href="#0078-NWLI-013">0078-NWLI-013</a>)
+* When the limit is reached in the market the [market data](./0021-MDAT-market_data_spec.md) API correctly indicates that. (<a name="0078-NWLI-023" href="#0078-NWLI-023">0078-NWLI-023</a>)
+* When the `limits.markets.maxPeggedOrders` limit is reached in the market, but `limits.markets.maxLPs` is not active yet it is still possible to successfully submit a new liquidity provision order (<a name="0078-NWLI-028" href="#0078-NWLI-028">0078-NWLI-028</a>).
+
+### LP order shapes
+
+* Submitting a [liquidity provision order](./0038-OLIQ-liquidity_provision_order_type.md) with 20 buy shapes and 20 sell shapes proceeds without any errors and liquidity provision becomes active. (<a name="0078-NWLI-014" href="#0078-NWLI-014">0078-NWLI-014</a>)
+* Submitting a liquidity provision order with 21 buy shapes and 20 sell shapes results in a failure. Error message attributes the rejection to the limit. (<a name="0078-NWLI-015" href="#0078-NWLI-015">0078-NWLI-015</a>)
+* Submitting a liquidity provision order with 20 buy shapes and 21 sell shapes results in a failure. Error message attributes the rejection to the limit. (<a name="0078-NWLI-016" href="#0078-NWLI-016">0078-NWLI-016</a>)
+
+### LPs
+
+* Attempt to place an additional liquidity provision with valid commitment amount below the lowest one in the market when the number of liquidity provisions in the market is equal to `limits.markets.maxLPs` results in a rejection. Error message attributes the rejection to the limit. (<a name="0078-NWLI-017" href="#0078-NWLI-017">0078-NWLI-017</a>)
```

Comment:
This will potentially need updates, depending on how the discussion above pans out.

Reply:
Committed your ELS suggestion and modified the ACs.